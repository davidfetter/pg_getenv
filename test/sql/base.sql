\set ECHO none
BEGIN;
\i sql/pg_getenv.sql
\set ECHO all

-- You should write your tests

SELECT pg_getenv();

ROLLBACK;
