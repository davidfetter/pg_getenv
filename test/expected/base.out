\set ECHO none
-- You should write your tests
SELECT pg_getenv('test');
 pg_getenv 
-----------
 
(1 row)

ROLLBACK;
